### PR TITLE
Electrs version bump (v0.9.10)

### PR DESCRIPTION
**Pull request by sutterseba**
#### What

Bumped electrs version numbers to v0.9.10, Issue https://github.com/raspibolt/raspibolt/issues/1147

I ran into a little error trying to checkout the new branch:

```
error: Your local changes to the following files would be overwritten by checkout:
	internal/config_specification.toml
Please commit your changes or stash them before you switch branches.
```

Maybe this is just me, but I cannot recall making any changes to that file so I added a little comment how to ignore something like this (simply by forcing with `-f`). 

#### Scope

- [ ] significant change to core configuration
- [ ] independent bonus guide
- [x] simple bug fix/core guide maintenance

### CODE REVIEW

File: guide/bitcoin/electrum-server.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ We get the latest release of the Electrs source code, verify it, compile it to a
...
+  Starting electrs 0.9.10 on aarch64 linux with Config { network: Bitcoin, db_path: "/data/electrs/db/bitcoin", daemon_dir: "/home/bitcoin/.bitcoin", daemon_auth: CookieFile("/home/bitcoin/.bitcoin/.cookie"), daemon_rpc_addr: 127.0.0.1:8332, daemon_p2p_addr: 127.0.0.1:8333, electrum_rpc_addr: 127.0.0.1:50001, monitoring_addr: 127.0.0.1:4224, wait_duration: 10s, jsonrpc_timeout: 15s, index_batch_size: 10, index_lookup_limit: Some(1000), reindex_last_blocks: 0, auto_reindex: true, ignore_mempool: false, sync_once: false, disable_electrum_rpc: false, server_banner: "Welcome to electrs 0.9.9 (Electrum Rust Server)!", args: [] }
```
Comment (VajraOfIndra) [inline guide/bitcoin/electrum-server.md:176]:
```suggestion
  Starting electrs 0.9.10 on aarch64 linux with Config { network: Bitcoin, db_path: "/data/electrs/db/bitcoin", daemon_dir: "/home/bitcoin/.bitcoin", daemon_auth: CookieFile("/home/bitcoin/.bitcoin/.cookie"), daemon_rpc_addr: 127.0.0.1:8332, daemon_p2p_addr: 127.0.0.1:8333, electrum_rpc_addr: 127.0.0.1:50001, monitoring_addr: 127.0.0.1:4224, wait_duration: 10s, jsonrpc_timeout: 15s, index_batch_size: 10, index_lookup_limit: Some(1000), reindex_last_blocks: 0, auto_reindex: true, ignore_mempool: false, sync_once: false, disable_electrum_rpc: false, server_banner: "Welcome to electrs 0.9.10 (Electrum Rust Server)!", args: [] }
```

Comment (VajraOfIndra) [reply guide/bitcoin/electrum-server.md:176]: LGTM, just a small missing change from 0.9.9 to 0.9.10 :)

Comment (sutterseba) [reply guide/bitcoin/electrum-server.md]: VS Code's "Change all occurrences" betraying me once again... :D

**Review by blckbx — APPROVED** [review]
tACK (on amd64)

Thanks! 🎉

P.S. Did not see the error you mentioned on `git checkout`. 
All went fine, no config modifications required updating from `v0.9.9`.

**Review by twofaktor — APPROVED** [review]
LGTM! utACK

**Comment by time-out** [comment]
The file /usr/local/bin/electrs now belongs 'admin'. Is that correct? I ask because electrs-old owner and group is root.

**Comment by blckbx** [comment]
> The file /usr/local/bin/electrs now belongs 'admin'. Is that correct? I ask because electrs-old owner and group is root.

Good point! Since we don't `sudo install` but rather `sudo cp` the binary to `/usr/local/bin` the owner and group is not set to `root`. Afaik it doesn't matter because we run electrs as systemd service pinned to user `electrs`. For the sake of uniformity, this could be improved by another short PR.

**Comment by sutterseba** [comment]
> For the sake of uniformity, this could be improved by another short PR.

Already included in  https://github.com/raspibolt/raspibolt/pull/1102. :)

**Pull request merged**